Snowflake.Data.Tests/SFMultiStatementsIT.cs

```diff
@@ -355,12 +355,12 @@ public void testWithAllQueryTypes()
 
                     // result of list
                     Assert.IsTrue(reader.NextResult());
-                    Assert.IsTrue(reader.HasRows);
+                    Assert.IsFalse(reader.HasRows); // no files staged for table t1
```

Comment:
Maybe we could think of splitting this as well into separated tests

Reply:
It tests multi statement SQL so it is difficult to split it into separated tests.

Reply:
My bad - your right